### PR TITLE
feature: web wallet was added

### PR DESCRIPTION
**Pull request by faisalnaveedRNS**
## Description

web wallet was added
Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-wallet-pylons/@next/components/molecules/history-details/history-details.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <Box
+      sx={{
+        minWidth: '300px'
+      }}
+    >
+      <div className="tab-panel" data-testid="historyDetails-molecule">
+        {history?.map((val, i) => (
+          <div className="item" key={i}>
+            <p>{moment(val.createdAt).format('DD/MM/YYYY hh:mm:ss')}</p>
```
Comment (kjawadDeveloper2) [inline web-wallet-pylons/@next/components/molecules/history-details/history-details.tsx:19]: This is the format we are using on the mobileend `MM/dd/yyyy`

File: web-wallet-pylons/@next/components/molecules/media-set/media-set.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+              height: '50px'
+            }}
+          >
+            <source src={src} type="audio/ogg" />
+            <source src={src} type="audio/mpeg" />
+            Your browser does not support the audio element.
+          </audio>
+        </div>
+      )
+    } else if (nftType.toLowerCase() === 'pdf') {
```
Comment (kjawadDeveloper2) [inline web-wallet-pylons/@next/components/molecules/media-set/media-set.tsx:72]: 'pdf' Can we create constants . As they are used in multi places.

File: web-wallet-pylons/@next/components/organisms/easel-buy-mob-view/easel-buy-mob-view.tsx [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+                    background: 'transparent',
+                    boxShadow: 'none',
+                    padding: '0px !important',
+                    width: '18rem'
+                  }}
+                >
+                  <MobOwnershipAccordionSummary
+                    expanded={expanded}
+                    accordionId="panel1"
+                    title="OwnerShip"
```
Comment (kjawadDeveloper2) [inline web-wallet-pylons/@next/components/organisms/easel-buy-mob-view/easel-buy-mob-view.tsx:244]: Can you change spelling from `OwnerShip` to Ownership.

File: web-wallet-pylons/@next/store/app.store.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+const persistedReducer = persistReducer(
+  persistConfig,
+  combineReducers({
+    auth: authReducer
+  })
+)
+export const createStore = configureStore({
+  reducer: persistedReducer,
+  middleware: (getDefaultMiddleware) =>
+    // eslint-disable-next-line implicit-arrow-linebreak
```
Comment (kjawadDeveloper2) [inline web-wallet-pylons/@next/store/app.store.ts:31]: Linter issue.

File: web-wallet-pylons/@next/store/slices/auth/auth-api.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { createAsyncThunk } from '@reduxjs/toolkit'
+import { requestUrl } from '@hooks'
+
+export const authenticateQR = createAsyncThunk(
+  'admin/AuthenticateQR',
+  async (
+    // eslint-disable-next-line @typescript-eslint/member-delimiter-style
```
Comment (kjawadDeveloper2) [inline web-wallet-pylons/@next/store/slices/auth/auth-api.ts:7]: linter.

File: web-wallet-pylons/@next/store/slices/auth/auth.slice.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/space-before-function-paren */
```
Comment (kjawadDeveloper2) [inline web-wallet-pylons/@next/store/slices/auth/auth.slice.ts:1]: linter.

File: web-wallet-pylons/@next/utils/get-blockchain-icon-path.ts [hunk 7]
```diff
@@ -0,0 +1,26 @@
+export const getCryptoCurrencyIcon = (crypto: string): string => {
+  switch (crypto?.toLowerCase()) {
+    case 'uatom':
```
Comment (kjawadDeveloper2) [inline web-wallet-pylons/@next/utils/get-blockchain-icon-path.ts:3]: Can we make constant.

File: web-wallet-pylons/@next/utils/get-blockchain-icon-path.ts [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+    case 'eeur':
+      return '/images/crypto_icons/eeur.svg'
+    case 'weth-wei':
+      return '/images/crypto_icons/eth.svg'
+    case 'uusd':
+    case 'ubedrock':
+    case 'umuon':
+    case 'ujunox':
+      return ''
```
Comment (kjawadDeveloper2) [inline web-wallet-pylons/@next/utils/get-blockchain-icon-path.ts:22]: Why we have nothing for ujuno?

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]
There are some images in the PR. can we remove this?

File: web-wallet-pylons/@next/hooks/use-axios-fetch.ts [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+}
+let tokenId: any
+
+export const useAxiosFetch: mainProps = (setSnackbarProps) => {
+  const [isLoading, setIsLoading] = useState(false)
+  const [fetchError, setFetchError] = useState<any>(null)
+  const [data, setData] = useState<any>()
+  const dispatch = useAppDispatch()
+  const router = useRouter()
+  // eslint-disable-next-line consistent-return
```
[the remark this answers is not in the document]
Comment (Kamran95) [reply web-wallet-pylons/@next/hooks/use-axios-fetch.ts:36]: Resolved. Some eslint rules are disabled as they might be conflicting with prettier settings

**Pull request closed without merging**